File: cpp/bmi.cpp
/*
 * bmi.cpp
 * 
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	float wzrost = 0;
    int masa = 0;
    float bmi = 0;
    
    cout<<"Podaj swoją masę: ";
    cin>>masa;
    cout<<"Podaj swój wzrost: ";
    cin>>wzrost;
    
    bmi = (masa / (wzrost*wzrost));
    cout<<endl<<bmi<<endl;
    
    
    if(bmi>=30)
    {
        cout<<"otyłość"<<endl;
    }
    else if(bmi>=25)
    {
        cout<<"nadwaga"<<endl;
    }
    else if(bmi>=18.5)
    {
        cout<<"norma"<<endl;
    }
    else
    {
        cout<<"niedowaga"<<endl;
    }
	return 0;
}

File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    wzrost = float(input("Podaj wzrost: "))
    masa = int(input("Podaj mase: "))
    bmi = masa / (wzrost * wzrost)
    print(bmi)

    if bmi >= 30:
        print("otyłość")
    elif bmi >= 25:
        print("nadwaga")
    elif bmi >= 18.5:
        print("norma")
    else:
        print("niedowaga")


    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
